tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -132,30 +149,86 @@ public function test_get_theme_item_permission_check() {
 		$this->assertErrorResponse( 'rest_cannot_manage_global_styles', $response, 403 );
 	}
 
-
 	/**
 	 * @covers WP_REST_Global_Styles_Controller::get_theme_item
 	 * @ticket 54516
+	 * @ticket 54596
 	 */
-	public function test_get_theme_item_invalid() {
+	public function test_get_theme_item_not_found() {
 		wp_set_current_user( self::$admin_id );
-		$request  = new WP_REST_Request( 'GET', '/wp/v2/global-styles/themes/invalid' );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/global-styles/themes/doesnotexist' );
```

Comment:
Why is this changing here?

Reply:
Carryover from testing theme that does not exist. Reverting.

Reply:
Reverted in https://github.com/WordPress/wordpress-develop/pull/2034/commits/03bf811e413e6e76d4a864dce0cda339dd930346.